doc/man/restic-forget.1

```diff
@@ -57,15 +57,15 @@ Exit status is 0 if the command was successful, and non\-zero if there was any e
 
 .PP
 \fB\-\-keep\-tag\fP=[]
-    keep snapshots with this \fB\fCtaglist\fR (can be specified multiple times)
+    keep snapshots with this \fB\fCtaglist\fR (can be specified multiple times). An empty tag list (--keep-tag='') will keep all untagged snapshots.
```

Comment:
Please remove the manpage changes. These are automatically regenerated before a new release.